upgrade-notes/backend_20240822_191848.md

```diff
@@ -0,0 +1,103 @@
+#### allow administrator to limit the managed domains ([#3289](https://github.com/shopsys/shopsys/pull/3289))
+
+-   interface `Shopsys\FormTypesBundle\Domain\DomainIdsProviderInterface` contains a new method `getAllLocales()`
+-   interface `Shopsys\FormTypesBundle\Domain\DomainIdsProviderInterface` now has the class `Shopsys\FrameworkBundle\Component\Domain\AdministratorSelectedDomainProvider` set as the service in the container
```

Comment:
I was struggling to understand this, maybe it would be nice to add something like  "instead of `Shopsys\FrameworkBundle\Component\Domain\Domain`"